### ISSUE TITLE
New term: increased spongiotrophoblast cell number

### ISSUE THREAD

**Issue by LaurensWilming** [mgijax/mammalian-phenotype-ontology#3620]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

**increased spongiotrophoblast cell number**

>2. Synonyms (e.g., absent spleen)

**higher spongiotrophoblast cell number**

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

**increase in the number of trophoblast cells that arise in the junctional zone of the placenta**

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

**PMID:34982814**

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

**MP:0008959:abnormal spongiotrophoblast cell morphology**

**More broadly: It may be worthwhile to create matching _increased-reduced_ (or _increased-decreased_) and _higher-lower_ pairs for all terms where only one such child exists. And the same for any other binary terms that may exist.**

**Comment by sbello** [comment]
We've been trying to pick these up as we come across them. Current policy is to add increased/decreased pairs but this wasn't always the case.
Thanks for catching this one.

**Comment by anna-anagnostop** [comment]
Added new term:
increased spongiotrophoblast cell number MP:0031431
